### DISCUSSION TITLE
GitHub Pages - CSS and JS 404 - SvelteKit

### DISCUSSION THREAD

**Discussion by Penca53** [community/community#52062] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have my [portfolio website](https://penca53.github.io/my-portfolio/) hosted on GitHub Pages. It is built with SvelteKit, SSG rendered, and deployed using gh-pages (npx). Today I made a minor change to the page and tried to deploy it, and all requests returned 404. I had the same problem a few months ago (Feb), but I solved them (it was due to .nojekyll file). 

Any ideas? Thanks.

**Comment by Penca53** [comment, marked as answer]
Turns out that the problem is that in my `svelte.config.js` I had 
`paths: {
	base: dev ? '' : '/my-portfolio',
  }`
because it was needed by GitHub in order to make it work. Apparently now it doesn't need it anymore, so all the 404 were right since it was trying to get non-existing files. 

Now I'm also using the Beta GitHub workflow deploy to Pages which is a lot easier and straightforward than the `gh-pages` npm package.

**Reply by Fasteroid** [reply]
Thanks so much, I copied your workflow and it saved my portfolio site!
Marked you as co-author to ensure I credit you: https://github.com/Fasteroid/svelte-portfolio/commit/7e06e7220795ceb41db265c6c55d8642718a5ea2

**Reply by Penca53** [reply]
Glad to hear that!

**Reply by phorjax** [reply]
Man, you just saved my life. I was fighting with this b*tch that didn't want to render anything for more than 4 hours. and removing the paths + using the new workflow deploy made the trick!. Thank you so much!

**Reply by BacelarVitor** [reply]
You are a life savior! thank you
